### PR TITLE
prepare for new parameterized test image job template

### PR DESCRIPTION
**Pull request by bparees**
depends on https://github.com/openshift/ci-tools/pull/24

### CODE REVIEW

File: ci-operator/config/openshift/cluster-authentication-operator/openshift-cluster-authentication-operator-master.yaml [hunk 1]
```diff
@@ -48,5 +48,6 @@ tests:
 - artifact_dir: /tmp/artifacts
   as: e2e-aws-console-login
   commands: ./test-prow-e2e.sh login
-  openshift_installer_console:
+  openshift_installer_custom_test_image:
     cluster_profile: aws
+    test_imagestream_tag: stable:console-tests
```
Comment (stevekuznetsov) [inline ci-operator/config/openshift/cluster-authentication-operator/openshift-cluster-authentication-operator-master.yaml]: I'd prefer `from` as the key name to parallel the container test type and would not expect people to need to specify `stable` -- that's an internal detail

File: ci-operator/templates/openshift/installer/cluster-launch-installer-console.yaml [hunk 2]
```diff
@@ -89,7 +85,7 @@ objects:
 
     # Once the cluster is up, executes shared tests
     - name: test
-      image: registry.svc.ci.openshift.org/${NAMESPACE}/${TEST_STREAM}:${TEST_TAG}
+      image: registry.svc.ci.openshift.org/${NAMESPACE}/${TEST_IMAGESTREAM_TAG}
```
Comment (stevekuznetsov) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-console.yaml]: this whole thing should be resolved locally with image lookup, no?

Comment (bparees) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-console.yaml]: hm?

File: ci-operator/config/openshift/cluster-authentication-operator/openshift-cluster-authentication-operator-master.yaml (hunk 1 above)
Comment (bparees) [reply ci-operator/config/openshift/cluster-authentication-operator/openshift-cluster-authentication-operator-master.yaml]: as you wish i guess, but i think my name choice offers more context to an inexperienced observer/reader of the file.

Comment (bparees) [reply ci-operator/config/openshift/cluster-authentication-operator/openshift-cluster-authentication-operator-master.yaml]: updated https://github.com/openshift/ci-tools/pull/24 w/ this change, and updated this PR to reflect it.

**Comment by bparees** [comment]
@stevekuznetsov i think you lied to me:

https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_release/4425/rehearse-4425-pull-ci-openshift-cluster-authentication-operator-release-4.2-e2e-aws-console-login/1

`template e2e-aws-console-login has required parameter TEST_IMAGESTREAM_TAG which is not defined`

I can see the env var defined on the ci-operator pod:
https://gist.github.com/bparees/87b79ac3bc2508923ccb905980e44a71#file-gistfile1-txt-L89

but ci-operator does not seem to have passed it to the template as a parameter.

**Comment by bparees** [comment]
(Clayton did confirm this is supposed to work, so i'll dig in some more on what dots i may have failed to connect)

**Comment by bparees** [comment]
@stevekuznetsov ok i'm stuck again..I do not see why this is failing to provide the required parameter:


```
template e2e-aws-console-login has required parameter TEST_IMAGESTREAM_TAG which is not defined
```

https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_release/4425/rehearse-4425-pull-ci-openshift-cluster-authentication-operator-release-4.2-e2e-aws-console-login/1

when i am clearly setting the env var here:
https://github.com/openshift/release/pull/4425/files#diff-c32b461cfc356aacc39cb76b517c2959R102

possible pj-rehearse bug?

**Pull request merged**